frontend/src/components/project/ProjectContent.js

```diff
@@ -318,38 +327,40 @@ export default function ProjectContent({
             {texts.created}: <DateDisplay date={new Date(project.creation_date)} />
           </Typography>
           <div>
-            <Typography component="span">
-              {texts.started + " "}
-              <TimeAgo
-                date={new Date(project.start_date)}
-                formatter={locale === "de" ? germanYearAndDayFormatter : yearAndDayFormatter}
-              />{" "}
-              {texts.by}
-            </Typography>
-            {project.isPersonalProject ? (
-              <MiniProfilePreview
-                className={classes.creator}
-                profile={project.creator}
-                size="small"
-              />
-            ) : (
-              <MiniOrganizationPreview
-                size="small"
-                className={classes.creator}
-                organization={project.creator}
-              />
-            )}
-            {project.end_date && (
-              <Typography className={classes.finishedDate}>
-                {texts.finished + " "}
+            <div className={classes.projectParentContainer}>
+              <Typography component="span">
+                {texts.started + " "}
                 <TimeAgo
-                  date={new Date(project.end_date)}
+                  date={new Date(project.start_date)}
```

Comment:
Just an observation here, but do you know why we have a mix of snake_case `start_date` props and camelCase `isPersonalProject`? I think in a future PR (definitely out of scope here!) it'd be great if we could refactor those for consistency.

Reply:
Wouldn't know why both cases are used, but I agree refactoring for consistency would be a nice change.